src/internal_events/stdin.rs

```diff
@@ -0,0 +1,45 @@
+use super::InternalEvent;
+use metrics::counter;
+
+#[derive(Debug)]
+pub struct StdinEventReceived {
+    pub byte_size: usize,
+}
+
+impl InternalEvent for StdinEventReceived {
+    fn emit_logs(&self) {
+        trace!(message = "received one event.", rate_limit_secs = 10);
```

Comment:
Should we have `rate_limit_secs` for `trace!` log? Currently only `stdin` and `file` use it.

Reply:
Not for tracing, in my opinion.